db/database.js

```diff
@@ -44,22 +50,25 @@ const createCohortsTable = db => {
 const createStudentsTable = db => {
   db.run(
     `CREATE TABLE students (
-      id INTEGER PRIMARY KEY AUTOINCREMENT,
+      student_id INTEGER PRIMARY KEY,
```

Comment:
replace auto-incrementing id with student_id. Will discuss if we want to have the auto-incrementing id

Reply:
Not sure why this change unless it's responding for the plans to have an id that we assign to students.  Even in that event, I'd use an auto-incrementing `id` field for joins and then just store "our" id as a second field.

What's the reasoning?

Reply:
Ok, I will make the change